src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -3481,14 +3481,14 @@ void Player_InitItemAction_SpawnHookshot(PlayState* play, Player* this) {
 }
 
 void Player_InitItemAction_5(PlayState* play, Player* this) {
```

Comment:
I'm going to leave this one alone because I have a follow-up PR naming this along with other init actions:
https://github.com/zeldaret/mm/compare/main...engineer124:mm:initAction